### ISSUE TITLE
Distinguish plasma_isolation and virus_isolation in subject_history

### ISSUE THREAD

**Issue by philiptzou** [hivdb/covid-drdb#27]
The plasma_isolation shouldn't have an `iso_name`. However virus_isolation should have. This is a temp solution before we can implement #11. With this temp solution we can at least apply stricter constraint.

**Comment by philiptzou** [comment]
No longer needed since #11 is resolved.
